components/CSVTable.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react'
+import Table from '@mui/material/Table'
+import TableBody from '@mui/material/TableBody'
+import TableCell from '@mui/material/TableCell'
+import TableContainer from '@mui/material/TableContainer'
+import TableHead from '@mui/material/TableHead'
+import TableRow from '@mui/material/TableRow'
+import Paper from '@mui/material/Paper'
+
+function createData (
+  email: string,
+  subject: string,
+  name: string,
+  company: string
+) {
+  return { email, subject, name, company }
+}
+
+const rows = [
+  createData('example@facebook.com', 'Sponsoring HackBeanpot', 'Danaerys', 'Facebook'),
+  createData('example@google.com', 'Sponsoring HackBeanpot', 'Jaime', 'Google'),
+  createData('example@appfolio.com', 'Sponsoring HackBeanpot', 'Tyrion', 'Appfolio')
+]
+
+export default function CSVTable () {
+  return (
+    <TableContainer component={Paper}>
+      <Table aria-label="simple table">
+        <TableHead>
+          <TableRow>
+          <TableCell align="left">email</TableCell>
+            <TableCell align="left">subject</TableCell>
+            <TableCell align="left">name</TableCell>
+            <TableCell align="left">company</TableCell>
```

Comment:
If we're creating a CSVTable component, in order for it to be the most useful in the future we shouldn't hard code the table cell values. Instead, we should pass in the values (let's say headers) like on line 25 and do something like headers.map(header => <TableCell align="left">{header}</TableCell>